### ISSUE TITLE
C-PORT Ships-in-Transit missing concentrations

### ISSUE THREAD

**Issue by arunacs** [CEMPD/ctools#154]
For some combination of source and pollutant, you don't have emissions, and hence no concentrations plotted. How do we make this information available upfront, without going through a submit, and then not seeing any concentrations displayed?

For e.g. Charleston has no Acetaldehyde, Butadiene and Diesel-PM

**Comment by michellegrace** [comment]
I am wondering if we should even have Diesel PM as a pollutant in C-PORT??.... this seems like a C-LINE pollutant and all other sources would have zero, since this is just the pm2.5MOVES emission for ONLY diesel vehicles, there is no distinction of fuel type for other sources. Catherine, can we hide this pollutant on C-PORT? ... should still be an option for C-LINE and the dispersion code, we just won't have the user be able to select it in C-PORT.

Ok that will fix 1/3 of this issue!

**Comment by arunacs** [comment]
Yes, I don’t believe we have ships identified by fuel type  (like diesel trucks vs gasoline cars for on-road). So, it should be fine to not have Diesel PM in C-PORT. The speciation factors to get Diesel-PM from PM25 emissions are the same, and the nomenclature varies based on the source fuel or SCC.


Sarav

**Comment by cseppan** [comment]
New issue #156 for web site.
